src/sqlancer/questdb/QuestDBSchema.java

```diff
@@ -70,6 +69,9 @@ public QuestDBCompositeDataType(QuestDBDataType dataType, int size) {
             case BOOLEAN:
                 isNullable = false;
                 break;
+            case SYMBOL:
+                isNullable = false;
```

Comment:
According to https://questdb.io/docs/reference/sql/datatypes/, the data type should be nullable, no?